src/client/src/rt-components/adaptive-loader/AdaptiveLoader.tsx

```diff
@@ -22,14 +23,17 @@ const getBounce = (moveDistance: number) => keyframes`
   }
 `
 
+const memoizedGetBounce = _.memoize(getBounce)
```

Comment:
Nit: you can just wrap the original and stick with `getBounce` — we expect ideal behavior from our functions, if the memoization is correct than we have no need to spread that knowledge.